kamailio/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "8ed5bcf6-21cb-4eae-86ff-11249b4212f2",
+  "app_id": "kamailio",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitoring agent for kamailio SIP server",
+    "title": "kamailio",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
+  },
+  "assets": {
```

Comment:
You should add your dashboard here

Reply:
resolved in cf85a7a2cd46cc4e6c399e639ff477102a24aead